test/lit/wasm-split/verbose.wast

```diff
@@ -0,0 +1,14 @@
+;; Test that --verbose mode correctly prints the kept and split funcs
```

Comment:
Can't we do this by adding one more command in an existing test, such as basic.wast?

Reply:
Oh, right, we can't handle multiple modules...

Reply:
Yes, and I use this functionality in the new test in #3400 as well. Do you think it would be a good idea to remove this test?

Reply:
Oh no, I was just wondering if this is the only thing we test in this file, probably we could've added this test in basic.wast by adding one more `RUN` line. But I don't feel strongly anyway and this looks fine as is.

Reply:
👍 (I fixed my email filters and am getting these notifications now, btw)